apps/sms/views/conversation/test/unit/recipients_test.js

```diff
@@ -143,6 +143,24 @@ suite('Recipients', function() {
       assertDeepEqual(recipient, fixture);
     });
 
```

Comment:
__typo:__ no need in trailing space in the test name, also maybe it will be better to have ```..... leading parenthesis ....```